tools/Custom/JsonExtensions.cs

```diff
@@ -1,8 +1,11 @@
 namespace NamespacePrefixPlaceholder.PowerShell.JsonUtilities
 {
     using Newtonsoft.Json.Linq;
+    using Newtonsoft.Json;
     using System;
     using System.Linq;
+    using System.Text.Json;
+    using System.Text.Json.Nodes;
```

Comment:
It doesn't looks like we're using `System.Text.Json` anywhere here. Any chance that's intended?

Reply:
Oh. Thanks for catching that, we don't need them. I've done the update